bots/namshubs/namshub_of_roadmap_updates.py

```
@@ -0,0 +1,188 @@
+    chain_workflow,
+    create_toolkit,
+    format_final_summary,
+    validate_required_params,
+)
+from bots.tools.code_tools import patch_edit, view, view_dir
+from bots.tools.python_execution_tool import execute_python
+from bots.tools.self_tools import branch_self
+from bots.tools.terminal_tools import execute_powershell
+
+
+def _set_roadmap_system_message(bot: Bot, pr_number: str = None) -> None:
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)